src/main/scala/io/corbel/sdk/http/CorbelHttpClient.scala

```diff
@@ -20,6 +20,9 @@ trait CorbelHttpClient extends HasConfig {
   object resources extends CorbelService {
     override def /(path: String)(implicit config: CorbelConfig): CorbelRequest = url(s"${config.resourceBaseUri}/$path")
   }
+  object notifications extends CorbelService {
+    override def /(path: String)(implicit config: CorbelConfig): CorbelRequest = url(s"${config.notificationsBaseUri}/$path")
```

Comment:
move the method `/` to CorbelService and make the baseUrl a class field. That way we don't have duplicate code